Topic 1/src/point6/User.java

```diff
@@ -0,0 +1,45 @@
+package point6;
+
+public class User {
+	private String name;
+	private String email;
+	private String password;
+	private int creditNumber;
+	
+	public User(String name, String email, String password, int creditNumber){
```

Comment:
[High] As *email*, *password* and *creditNumber* should be optionals, there should not be part of constructor.
Or at least, you should have a constructor with only *required* properties.